libs/nucs/src/validator.rs

```diff
@@ -392,6 +391,8 @@ impl fmt::Display for ValidationKind {
             RootKeySignatureMissing => "root NUC is not signed by root keypair",
             SubjectNotInChain => "subject not in chain",
             TokenExpired => "token is expired",
+            TooManyProofs => "up to one `prf` in a token is allowed",
```

Comment:
nit: Have we used the acronym `prf` elsewhere? PRF usually stands for pseudorandom function and it could be slightly confusing if we use it for proof.

Reply:
yup, makes sense. Maybe we can refactor this at some point but it is not urgent.

Reply:
UCANs also use `prf` so I don't know if we should deviate just to avoid an abbreviation. JWTs contain abbreviations for all fields already anyway, e.g. `iss` is a standard field (not the international space station!)